### ISSUE TITLE
Visual bug: Chat field/input items (sigil, hint text, "send code" button) are not vertically centered in field

### ISSUE THREAD

**Issue by urcades** [urbit/urbit#3033]
Refer to image below:

<img width="1130" alt="Screen Shot 2020-06-19 at 11 24 55 AM" src="https://user-images.githubusercontent.com/1195363/85150856-11c4da80-b221-11ea-9055-9d5e60f965a0.png">

Simply, these items should be vertically centered in the field in their default state. When the field grows (shown in the below image), element positioning works as originally designed:

<img width="1138" alt="Screen Shot 2020-06-19 at 11 37 07 AM" src="https://user-images.githubusercontent.com/1195363/85151101-5c465700-b221-11ea-9273-78dffe2a9701.png">

Bug is observed in Safari `Version 13.1.1 (15609.2.9.1.2)` and to a lesser extent in Chrome `Version 83.0.4103.106 (Official Build) (64-bit)` (in chrome, the sigil and input text are centered, but the 'send code' and 'send image' buttons are misaligned).

**Comment by matildepark** [comment]
@urcades are we ... any better with this now? After #3067 I think this lined up.

**Comment by matildepark** [comment]
Poke @urcades, can this be closed

**Comment by urcades** [comment]
Yes, this is all fixed up in its current state!

**Comment by urcades** [comment]
Commenting here with another related detail I brought up in our other chat, but I was wondering if it was worthwhile to actually reverse the behavior of the chat input's buttons such that the sigil and action buttons 'stick' to the bottom. 

I get the general sense that this behavior, compared to the current behavior seen in the screenshot above (chat buttons and sigil move upward as the input grows), will generally feel more perceptually 'solid', with the interface moving around less.

An example recorded via Figma:

![input](https://user-images.githubusercontent.com/1195363/90568533-156dd280-e17a-11ea-8c8c-a2795410c7bf.gif)

If this ends up being clunkier to implement, or requires a more complex implementation, it may not be worth it, but if it's easy to implement, I'd prefer the input works as it does in the provided example.

**Comment by matildepark** [comment]
Out of scope of this but PR incoming!
